flash/vision/classification/components/model_zoo.py

```diff
@@ -0,0 +1,20 @@
+import torchvision
+
+TORCHVISION_MODEL_ZOO = {
```

Comment:
do we need this?

why not do instead `getattr(torchvision.models, backbone, None)`

Reply:
Because not supporting all [torchvision](https://pytorch.org/vision/stable/models.html) models for nor

Reply:
I don't think we necessarily need to support all of the torchvision models, and each one requires separate logic to separate the "backbone" portion of the model

Reply:
Yup

Reply:
But we can still do this filtering in `torchvision_backbone_and_num_features`

And raise MisconfigurationException if unsupported

Reply:
We are calling `torchvision` before the filtering part. Also, I felt it was better to have the config/collection? decoupled from the code logic.

Reply:
It's not really decoupled because you still have

```python
if model_name in ...

elif model_name in ...

elif model_name in ...
```

Reply:
so might as well have

```python
model = getattr(torchvision.models, model_name, None)
if model is None:
    raise MisconfigurationException
if model_name in ...

elif ...

```

and avoid having to maintain the list of models in both places

Reply:
Makes sense, updated! @carmocca